source/algorithms.tex

```diff
@@ -5190,7 +5190,7 @@
 \pnum
 \complexity
 At most
-$\max(\lfloor{\frac{3}{2}} (N-1)\rfloor, 0)$
+$\max(\left\lfloor{\frac{3}{2}} (N-1)\right\rfloor\mathclose{}, 0)$
```

Comment:
What happens if you just use `\bigl`, `\bigr`? The advantage of those is that they are guaranteed to fit into the line, whereas the stretchable versions may end up taller than a line.